src/Configuration/CFGCompiler/CFGCompiler.cpp

```diff
@@ -106,9 +106,11 @@ bool CFGCompiler::RegisterCallbackFunction(std::string name,
 int CFGCompiler::Compile(CFGCompiler* cfgcompiler, bool batchMode) {
   // Set generic information that every command might need
   Compiler* compiler = cfgcompiler->GetCompiler();
-  cfgcompiler->m_cmdarg.projectName = compiler->ProjManager()->projectName();
```

Comment:
@guanyung-rs why do we need this ? In case of a standalone application, we won't have project etc. defined so this needs to be handled properly.

Reply:
I check the code. I believe this code is not required or being used in programmer module.